src/i18n/index.js

```diff
@@ -66,6 +68,25 @@ export const i18nOptions = {
           return numeral(param).format(
             i18n.config.number.formats.integer
           )
+        case 'percent':
+          return numeral(param).format(
+            i18n.config.number.formats.percent
+          )
+        case 'fee_type':
+          const feeTypeIndex = Object.values(FEE_TYPES).indexOf(param)
```

Comment:
let's leave the logic as it was before, when we had a `[feeType]translation` map. the only change I requested is using the keys not as a primitive: 

```
{
  0: 'fee-types.paymentFee'
}
```

but the calculated value:

```
{
  [FEE_TYPES.paymentFee]: 'fee-types.paymentFee'
}
```

Reply:
and you also have to write the unit test on the formatter and add it's API to the README

Reply:
Resolved, updated